main/networking/Runtime.ts

```diff
@@ -305,6 +306,13 @@ class TCPConn {
           case MsgType.CHALLENGE_DATA:
             // TODO: Dispatch challenge outputs to redux
             break;
+          case MsgType.RUNTIME_STATUS:
+            decoded = protos.RuntimeStatus.decode(packet.payload);
+            //Not sure if we need to update runtimeStatus state bcs heartbeat keeps track of that already
+            RendererBridge.reduxDispatch(updateCodeStatus(decoded.mode));
```

Comment:
Runtime Status sends us the current run mode it is in. Calls the updateCodeStatus action to change the mode